x-pack/test/api_integration/apis/maps/get_tile.js

```diff
@@ -29,7 +29,7 @@ export default function ({ getService }) {
 
       const jsonTile = new VectorTile(new Protobuf(resp.body));
       const layer = jsonTile.layers[MVT_SOURCE_LAYER_NAME];
-      expect(layer.length).to.be(2);
+      expect(layer.length).to.be(3); // 2 docs + the metadata feature
```

Comment:
Can you add an expect clause that the tile meta feature contains the expected output like properties and bounds?

Reply:
thx. adding these tests revealed an issue with the metadata-tile being included twice for doc-tiles. I fixed that as well